hasjob/models/domain.py

```diff
@@ -52,6 +53,15 @@ def use_title(self):
     def has_profile(self):
         return bool(self.title and self.description)
 
+    @cached_property
+    def get_logo_url(self):
```

Comment:
Since `logo_url` is itself a property, maybe something like `effective_logo_url`